frontend/src/app/main/ui/workspace/sidebar/options/menus/shadow.scss

```diff
@@ -34,6 +35,20 @@
 
 .shadow-element {
   @include flexColumn;
+
+  position: relative;
+
+  &:hover .reorder-icon {
+    visibility: visible;
+  }
+
+  &.dnd-over-top .reorder-separator-top {
+    display: block;
+  }
+
+  &.dnd-over-bot .reorder-separator-bottom {
+    display: block;
+  }
```

Comment:
pls avoid CSS selectors with high specificity 🙏 Details in our [UI Guidelines](https://help.penpot.app/technical-guide/developer/ui/#favor-lower-specificity)

Reply:
But I need it to get the ":hover" (and the class applied to the parent). I can move the selector to the other one but the "compiled" specify is the same.

Reply:
I'll change it so it passes a property instead of using nested selectors.